routes/search/resolvers.js

```diff
@@ -0,0 +1,82 @@
+const docsEn = require('../../lib/i18n').docs['en-US']
```

Comment:
Something to ponder: how hard would it be to serve up localized search results? I know that the repos and npm packages are not going to change from English. but the docs content might be fairly easy to implement? (Famous last words)